nucypher/policy/conditions/time.py

```diff
@@ -15,9 +15,12 @@ class TimeCondition(ReencryptionCondition):
     METHOD = 'timelock'
 
     class Schema(CamelCaseSchema):
-        name = fields.Str()
-        method = fields.Str(dump_default="timelock")
-        return_value_test = fields.Nested(ReturnValueTest.ReturnValueTestSchema())
+        SKIP_VALUES = (None,)
+        name = fields.Str(required=False)
```

Comment:
Perhaps we can implement `name` in a shema baseclass which can eventually also include `condiitonType`.

Reply:
There is something defined in the base Schema class here - https://github.com/nucypher/nucypher/blob/development/nucypher/policy/conditions/base.py#L53.

Let's do that when we actually utilize `name` for the future feature you mentioned. I was really just doing the most basic functionality for allowing for it to be allowed from the json Schema.